packages/azure-themes/src/azure/styles/DefaultButton.styles.ts

```diff
@@ -1,51 +1,79 @@
+      fontSize: FontSizes.size13,
+      border: `${StyleConstants.borderWidth} solid ${extendedSemanticColors.buttonBorderColor}`,
+      color: semanticColors.buttonText,
+      padding: '0 16px',
+      height: '24px',
+      selectors: {
+        // standard button
+        '&.ms-Button--default.ms-Button--hasMenu.is-expanded': {
+          borderColor: extendedSemanticColors.buttonBorderColor,
+          backgroundColor: semanticColors.buttonBackgroundPressed,
+          color: semanticColors.buttonText,
+        },
+      },
     },
     rootDisabled: {
       backgroundColor: semanticColors.buttonBackgroundDisabled,
       color: semanticColors.buttonTextDisabled,
       border: `0px`,
     },
     rootHovered: {
-      border: `${StyleConstants.borderWidth} solid ${semanticColors.buttonTextHovered}`,
+      color: semanticColors.buttonText,
+      fill: semanticColors.buttonText,
+      backgroundColor: extendedSemanticColors.buttonBackgroundHoveredDefault,
+      border: `${StyleConstants.borderWidth} solid ${extendedSemanticColors.buttonBorderHovered}`,
     },
     rootPressed: {
-      border: `${StyleConstants.borderWidth} solid ${semanticColors.buttonTextPressed}`,
+      borderColor: extendedSemanticColors.buttonBorderColor,
+      backgroundColor: semanticColors.buttonBackgroundPressed,
+      color: semanticColors.buttonText,
+    },
+    rootFocused: {
+      borderColor: extendedSemanticColors.buttonBorderColor,
+      color: semanticColors.buttonText,
     },
     rootChecked: {
-      border: `${StyleConstants.borderWidth} solid ${semanticColors.buttonTextPressed}`,
+      border: `${StyleConstants.borderWidth} solid ${extendedSemanticColors.buttonBorderColor}`,
+      backgroundColor: semanticColors.buttonBackgroundPressed,
+      color: semanticColors.buttonText,
     },
     rootCheckedHovered: {
-      backgroundColor: semanticColors.buttonBackgroundHovered,
-      color: semanticColors.buttonTextHovered,
+      color: semanticColors.buttonText,
+      fill: semanticColors.buttonText,
+      backgroundColor: extendedSemanticColors.buttonBackgroundHoveredDefault,
+      border: `${StyleConstants.borderWidth} solid ${extendedSemanticColors.buttonBorderHovered}`,
     },
     rootCheckedPressed: {
-      border: `${StyleConstants.borderWidth} solid ${semanticColors.buttonTextPressed}`,
-      color: semanticColors.buttonTextPressed,
+      borderColor: extendedSemanticColors.buttonBorderColor,
+      backgroundColor: semanticColors.buttonBackgroundPressed,
+      color: semanticColors.buttonText,
     },
     splitButtonContainer: {
       selectors: {
         // standard button
         '.ms-Button--default': {
-          border: `${StyleConstants.borderWidth} solid ${semanticColors.buttonText}`,
-          borderRight: 'none',
+          border: `${StyleConstants.borderWidth} solid ${extendedSemanticColors.buttonBorderColor}`,
+          borderRight: '0 !important',
```

Comment:
are we sure we want to use !important here?

Reply:
I felt this was necessary for the split button, as there should be no "inner" border.